docs/advanced-topics/advanced-losses-optimizers-and-poolers.md

```diff
@@ -78,6 +78,12 @@ Each color represents a different class. You can see how all of the loss functio
 but struggle to separate the green, blue, pink, purple and red classes,
 with `TripletMarginLoss` sperarating them the least, and `ArcFaceLoss` separating them the most.
 
+### CosineSimilarityLoss
+
+`CosineSimilarityLoss` is a regression loss function,
+which is calculated by comparing the cosine similarity of two embeddings against their ground truth cosine similarity.
```

Comment:
```suggestion
which is calculated by comparing the cosine similarity of two embeddings against their ground-truth cosine similarity or some other numerical measure of similarity in the range of 0.0 (completely different) to 1.0 (identical). 
```

In your examples, you emphasize that it's a similarity metric, not a ground-truth measure of cosines.